financial-connections/build.gradle

```diff
@@ -33,6 +33,7 @@ dependencies {
     implementation libs.compose.activity
     implementation libs.compose.foundation
     implementation libs.compose.material
+    implementation libs.compose.materialIcons
```

Comment:
Idk why this started not compiling, but we didn't have this dependency in these modules, but we were using the dependencies.